api/bonsai_api/redis/minhash.py

```diff
@@ -6,7 +6,8 @@
 from rq import Retry
 from rq.job import Dependency
 
-from . import ClusterMethod, SubmittedJob, TypingMethod
+from ..models.cluster import TypingMethod
```

Comment:
For me as a reader I would prefer full imports (rather than having to think about the exact current location)

Seems even more so for these `..` imports